lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/sim/rest/event/impl/ScoreUpdateEvent.java

```diff
@@ -51,32 +49,42 @@
 	 * The new session score of the user
 	 */
 	
-	public ObjectNode updatedScore;
-
+	private ScoreType scoreUpdateType;
+	private Float scoreUpdateValue;
```

Comment:
For messages, we adopted the convention to set all the fields visibility to `public`, as they are pure data objects.
Better changing the visibility of these two accordingly :wink: